jest.config.js

```diff
@@ -8,6 +10,7 @@ module.exports = {
   transform: {
     "^.+\\.(js|jsx|ts|tsx|mjs)$": ["babel-jest"],
   },
+  transformIgnorePatterns: [`/node_modules/(?!${esModules})`],
```

Comment:
for some reason jest + babel doesn't like the wallet-sdk, so I had to add this line. Otherwise would get a "can't use import statement" error

Reply:
Hm, interesting. I can take a look into this, but def not a blocker. This solution looks fine to me!